Packs/FireEyeNX/ReleaseNotes/1_1_10.md

```diff
@@ -1,10 +1,3 @@
-<<<<<<< HEAD
-#### Parsing Rules
-##### New: FireEyeNX Parsing Rule
-- Added parsing rule for _time extraction (Available from Cortex XSOAR 6.10.0).
```

Comment:
Is this mentioned in a different release note?

Reply:
I checked
it is not mentioned
and what I deleted is mentioned in RN 1.1.11